src/test/java/org/apache/commons/io/FilenameUtilsTest.java

```diff
@@ -251,6 +263,8 @@ public void testGetExtension() {
         assertEquals("", FilenameUtils.getExtension("a/b/c"));
         assertEquals("", FilenameUtils.getExtension("a.b\\c"));
         assertEquals("txt", FilenameUtils.getExtension("a.b\\c.txt"));
+        assertEquals("~", FilenameUtils.getExtension("a.b\\c.~"));
```

Comment:
Now this is a test case I would never had thought about :smile: great job @wodencafe !

Reply:
Thank you @kinow , happy to help :grin: